02-Reproject-Raster-In-R.Rmd

```diff
@@ -194,32 +205,34 @@ crs(DTM_hill_HARV)
 extent(DTM_hill_UTMZ18N_HARV)
 extent(DTM_hill_HARV)
 
-
 ```
 
 Notice in the output above that the CRS of `DTM_hillUTMZ18N_HARV` is now UTM. 
 However, the extent of the object is different. 
+
 Why do you think this is?  
 
+Note: When you are reprojecting a raster, you are moving it from one "grid" 
+to another. Thus you are modifying the data! Keep this in mind as you work with
+raster data. {: .notice2}
+
+
+##Assigning Resolution to Raster
```

Comment:
this is dangerous language. :) we don't want to "Assign resolution"

How about ##Dealing with Raster Resolution